client/my-sites/checkout/composite-checkout/wpcom/components/wp-checkout.js

```diff
@@ -127,8 +130,46 @@ export default function WPCheckout( {
 		setShouldShowContactDetailsValidationErrors,
 	] = useState( false );
 
+	const emailTakenLoginRedirectMessage = ( emailAddress ) => {
+		const redirectTo = '/checkout/no-site?cart=no-user';
+		const isNative = config.isEnabled( 'login/native-login-links' );
+		const loginUrl = login( { redirectTo, emailAddress, isNative } );
+		const loginRedirectMessage = translate(
+			'Choose a different email address. This one is not available. If this is you {{a}}log in now{{/a}}.',
```

Comment:
Minor, but I think the wording of this message could be improved. Maybe, "That email address is already in use. If you have an existing account, {{a}}please log in{{/a}}."

Reply:
Updated copy.